guides/WRITING-DOCUMENTATION.md

```diff
@@ -2,25 +2,79 @@
 
 Documentation for the design system's components are automatically generated from their CSS and JS comments. In order for this to work, you need to follow some conventions set by the tools we use to generate the documentation: [KSS](https://github.com/kss-node/kss-node) and [react-docgen](https://github.com/reactjs/react-docgen).
 
-## File naming
+- [Directory structure](#directory-structure)
+  - [The docs folder structure](#the-docs-folder-structure)
+  - [The src folder structure](#the-src-folder-structure)
+- [Component documentation page formatting](#component-documentation-page-formatting)
```

Comment:
I'm reading through this page structure and it feels like it could be clarified a bit. "Component documentation page formatting" vs "Documenting React components" sounds a bit confusing to me. I wonder if we can reorganize this document on a high level to be something like this:

```
- Folder structure
- Writing component documentation
   - Intro to KSS, when to use it
   - flags, markup, styleguide, etc
   - example
- Writing general documentation
   - front-matter
   - properties
   - example
```